symbiotic-build.sh

```diff
@@ -532,7 +534,7 @@ if [ $FROM -le 7 ]; then
 		include/symbiotic.h \
 		lib.c \
 		memalloc.c \
-		LLVM_SLICER_VERSION \
+		#LLVM_SLICER_VERSION \
```

Comment:
I think it is appropriate to remove it later, with the other commented out lines regarding old slicer.
Moving it to the end of the list.